prep-css-flexbox/display-flex.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - display: flex</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    .star-wars-card {
+      padding: 0.5rem;
+      border: 1px solid lightgray;
+      border-radius: 0.25rem;
+      box-shadow: 0 8px 8px -8px rgb(0 0 0 / 50%);
+      flex-basis: 25%;
+    }
+
+    .star-wars-card img {
+      width: 100%;
+      background-color: lightgray;
+    }
```

Comment:
Hi Anthony, you should be able to remove this chunk of code from all of your HTML files since it is written in `starwars.css` and linked in the `<head>` of each document already.